NestedAccordion/js/scripts.js

```diff
@@ -0,0 +1,34 @@
+// the interactive elements:
+// blue words have the UI accordion drop down with more information.
+// the arrows are light grey until you hoover over them.
+//
+// when i click on a heading, the text below hides and shows.
```

Comment:
Good start! As shown in class, work on getting everything down to one line and break it into baby steps.